config/config.go

```diff
@@ -202,6 +202,10 @@ type StorageConfig struct {
 
 	// Backend is the storage backend to select.
 	Backend string `koanf:"backend"`
+
+	// If true, we'll first delete all tables in the DB to
+	// force a full re-index of the blockchain.
+	WipeStorage bool `koanf:"DANGER__wipe_storage_on_startup"`
```

Comment:
I'd be cool with all caps too.

Reply:
I take it that's your preference? Sure, I'm fine either way. Uppercased.

Reply:
😬 soft preference, but yes, you called it.